draft-homburg-deleg-incremental-deleg.md

```diff
@@ -549,6 +549,8 @@ If the incremental deleg does not exist, then it is simply absent from the autho
 ## Resolver behavior with authoritative name server support {#behavior-with-auth-support}
 
 Incremental deleg supporting authoritative name servers will include the incremental delegation information (or the NSEC(3) records showing the non-existence) in the authority section of referral responses.
+For an unsigned zone, an incremental deleg supporting authoritative cannot return that an incremental delegation is absent (because of lack of an authenticated denial of existence), however with support from the served zone (the zone has the Resource Record provisioned `*._deleg IN IDELEG 0 .`), the authoritative name server can signal support also for unsigned zones (see {{extra-optimized (Extra optimized implementation)}}).
+
 If it is known that an authoritative name server supports incremental deleg, then no direct queries for the incremental delegation need to be sent in parallel to the legacy delegation query.
 A resolver SHOULD register that an authoritative name server supports incremental deleg when the authority section, of the returned referral responses from that authoritative name server, contains incremental delegation information.
```

Comment:
Maybe we should add:

If a resolver has registered that an authoritative name server supports incremental deleg and resolver receives a delegation without (denial of) an incremental delegation then resolver MUST issue a separate incremental deleg query and MAY register that authoritative server does not support incremental deleg.

Reply:
Agree, I'll add it

Reply:
Wait a minute. It already says this in the last two paragraphs of this subsection

Reply:
@Philip-NLnetLabs can you confirm your suggestion was already in the text?

Reply:
The part that I wanted to add was 'and MAY register that authoritative server does not support incremental deleg.' I assumed, but did not verify that the other part about sending a separate query was already there. But felt that a bit of redundancy would be warranted in this case.

Reply:
Well the redundency is very close to each other. I'll see if I can extend already existing paragraph with 'and MAY register that authoritative server does not support incremental deleg.'

Reply:
That is fine with me